src/interfaces/HubPool.ts

```diff
@@ -50,6 +50,11 @@ export interface ExecutedRootBundle extends SortableEvent {
   proof: string[];
 }
 
+export type TokenRunningBalance = {
+  runningBalance: BigNumber;
+  incentiveBalance: BigNumber;
+};
+
```

Comment:
Was this exposed in a previous PR?

Reply:
Kinda. I initially implemented it here, then realised you were also defining something similar and that we should share the definition.

This PR is now dependent on the PR where this actually will be defined, so it should disappear from here. (I think github is being weird but not realising that it's the same change).